packages/tex-to-myst/src/figures.ts

```diff
@@ -45,18 +46,33 @@ const FIGURE_HANDLERS: Record<string, Handler> = {
     state.closeParagraph();
     const url = texToText(getArguments(node, 'group'));
     const args = getArguments(node, 'argument')?.[0]?.content ?? [];
+    const params = extractParams(args);
+
+    // Only support width and page for now
+    for (const key in params) {
+      if (key !== 'width' && key !== 'page') {
+        delete params[key];
+      }
+    }
+
     // TODO: better width, placement, etc.
-    if (
-      args.length === 4 &&
-      args[0].content === 'width' &&
-      args[1].content === '=' &&
-      Number.isFinite(Number.parseFloat(args[2].content))
-    ) {
-      const width = `${Math.round(Number.parseFloat(args[2].content) * 100)}%`;
-      state.pushNode(u('image', { url, width }));
-    } else {
-      state.pushNode(u('image', { url }));
+
+    // Convert width to percentage if present
+    if (params.width) {
+      if (typeof params.width === 'number') {
+        params.width = `${Math.round(params.width * 100)}%`;
+      } else {
+        delete params.width; // If width is a string, we don't know what it is, so we ignore it
+      }
+    }
+    if (params.page) {
+      if (typeof params.page === 'number') {
+        params.page = Number(params.page) - 1; // Convert to 0-based for imagemagick
```

Comment:
Do we need to round or parse this or have something similar to `Number.isFinite(Number.parseFloat(params.page))`?